packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -12,33 +12,35 @@ import { dateI18n, getDate, humanTimeDiff, getSettings } from '@wordpress/date';
 import { store as coreStore } from '@wordpress/core-data';
 import { useSelect } from '@wordpress/data';
 
+const DAY_IN_MILLISECONDS = 60 * 60 * 1000 * 24;
+const dateFormat = getSettings().formats.date;
+const datetimeAbbreviatedFormat = getSettings().formats.datetimeAbbreviated;
+
 /**
  * Returns a button label for the revision.
  *
- * @param {Object} revision A revision object.
+ * @param {string|number} id                A revision object.
+ * @param {boolean}       isLatest          Whether the revision is the most current.
+ * @param {string}        authorDisplayName Author name.
+ * @param {Date}          modifiedDate      Revision modified date object.
  * @return {string} Translated label.
  */
-function getRevisionLabel( revision ) {
-	const authorDisplayName = revision?.author?.name || __( 'User' );
-
-	if ( 'parent' === revision?.id ) {
+function getRevisionLabel( id, isLatest, authorDisplayName, modifiedDate ) {
```

Comment:
Tiny nit: would it be worth switching over to an object as a param so that it's easier to add or remove elements for the label rather than use positional params?

Reply:
Nothing against that approach. The function is internal to the component so I'm not sure of the value aside from making refactoring easier later.

Reply:
Feel free to leave it if you like, since it's not exposed anywhere 🙂